server/tests/backends/pandas_executor/test_pandas_executor_steps.py

```diff
@@ -0,0 +1,64 @@
+import json
+from glob import glob
+from os import path
+
+import pandas as pd
+import pytest
+
+from tests.utils import assert_dataframes_equals
+from weaverbird.backends.pandas_executor import execute_pipeline
+from weaverbird.pipeline import Pipeline
+
+fixtures_dir_path = path.join(path.dirname(path.realpath(__file__)), '../fixtures')
+step_cases_files = glob(path.join(fixtures_dir_path, '*/*.step.json'))
+
+test_cases = []
+for x in step_cases_files:
+    case_hierarchy = path.dirname(x)[len(fixtures_dir_path) :]
+    case_name = path.splitext(path.basename(x))[0].split('.')[0]
+    case_id = case_hierarchy + '_' + case_name
+
+    # Path of the file specifying the step to test (xxx.step.json)
+    step_file_path = x
+
+    # Path of the file specifying the input dataframe (xxx.in.json)
+    in_file_path = path.join(path.dirname(x), f'{case_name}.in.json')
+
+    # Combination test requires other dataframes (xxx.in.domain_name.json)
+    other_in_files = glob(path.join(path.dirname(x), f'{case_name}.in.*.json'))
+    # Extract their domain name and map them to their file path ({ domain_name: 'path/xxx.in.domain_name.json' })
+    other_in_files_domains = {
+        path.splitext(path.basename(oif))[0].split('.')[-1]: oif for oif in other_in_files
+    }
+
+    # Path of the file specifying the expected output dataframe (xxx.out.json)
+    out_file_path = path.join(path.dirname(x), f'{case_name}.out.json')
+
+    test_cases.append(
+        pytest.param(
+            case_id, step_file_path, in_file_path, other_in_files_domains, out_file_path, id=case_id
+        )
+    )
```

Comment:
This will be to extract so that other backends test runners can use it

Reply:
:ok_hand: